.cirrus.yml

```diff
@@ -195,25 +195,26 @@ task:
 task:
   name: macOS
   macos_instance:
-    image: ghcr.io/cirruslabs/macos-sonoma-base:latest
+    image: ghcr.io/cirruslabs/macos-runner:sonoma
   env:
     HAVE_IPV6_LOCALHOST: yes
     USE_SUDO: true
+    CPPFLAGS: -I/opt/homebrew/opt/openssl@3/include
+    LDFLAGS: -L/opt/homebrew/opt/openssl@3/lib
   setup_script:
     - brew install autoconf automake bash libevent libtool openssl pandoc pkg-config postgresql@${PGVERSION}
     - python3 -m venv venv
     - venv/bin/pip install -r requirements.txt
     - echo 'anchor "pgbouncer_test/*"' | sudo tee -a /etc/pf.conf
     - sudo pfctl -f /etc/pf.conf
-  env:
-    CPPFLAGS: -I/opt/homebrew/opt/openssl@3/include
-    LDFLAGS: -L/opt/homebrew/opt/openssl@3/lib
   build_script:
     - ./autogen.sh
     - ./configure --prefix=$HOME/install --enable-werror
     - make -j4
+  env:
+    PATH: /opt/homebrew/opt/postgresql@${PGVERSION}/bin:$PATH
   test_script:
-    - source venv/bin/activate && make check -j4 CONCURRENCY=4
+    - source venv/bin/activate && make -j4 check CONCURRENCY=4
```

Comment:
Was this change actually needed?

Reply:
There is no strong reason. Consistency with the similar commands. I should have mentioned it.